### PR TITLE
Organize Terraform Configuration workshop

### PR DESCRIPTION
**Pull request by robin-norwood**

### CODE REVIEW

File: terraform-organize-configuration-workshop/courseBase.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+mkdir -p learn-terraform/assets
+
+cp index.html learn-terraform/assets/
```
Comment (topfunky) [inline terraform-organize-configuration-workshop/courseBase.sh:3]:
```suggestion
cp /root/learn-terraform/index.html /root/learn-terraform/assets/
```
Use full paths for reliability of you are using the assets feature.

File: terraform-organize-configuration-workshop/index.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    "intro": {
+      "text": "intro.md",
+      "code": "courseBase.sh",
+      "credits": "https://learn.hashicorp.com/terraform"
+    },
+    "finish": {
+      "text": "finish.md",
+      "credits": "https://learn.hashicorp.com/terraform"
+    },
+    "assets": {
```
Comment (topfunky) [inline terraform-organize-configuration-workshop/index.json:39]: The assets feature is flaky. I would clone a git repo instead.

File: terraform-organize-configuration-workshop/step2.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+#   region     = "us-west-2"
+# }
+
+# resource "random_pet" "petname" {
+#   length    = 4
+#   separator = "-"
+# }
+```
+
+Also remove the resource blocks for your dev environment from `prod.tf{{open}}:
```
Comment (topfunky) [inline terraform-organize-configuration-workshop/step2.md:38]:
```suggestion
Also remove the resource blocks for your dev environment from `prod.tf`{{open}}:
```

File: terraform-organize-configuration-workshop/step5.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+Now do the same with your prod environment.
+
+The steps to update and apply your production configuration are nearly identical
+to the ones for your dev environment. Be sure to apply your configuration with
+the `-var-file=prod.tfvars` flag.
+
+## Destroy Resources
+
+Clean up both environments by running `terraform destroy` in both directories.
```
Comment (topfunky) [inline terraform-organize-configuration-workshop/step5.md:174]:
```suggestion
Clean up both environments by running `terraform destroy`{{execute}} in both directories.
```
These should be listed as clickable commands for both directories. Destruction is important, especially if they are using our AWS account.

File: terraform-organize-configuration-workshop/index.json (hunk 2 above)
Comment (robin-norwood) [reply terraform-organize-configuration-workshop/index.json:39]: +1, will do after beta teach

**Pull request merged**